python/lsst/jointcal/dataIds.py

```diff
@@ -90,7 +87,6 @@ def makeDataRefList(self, namespace):
                     # Going with just the nearest tract.  Since we're throwing all tracts for the visit
                     # together, this shouldn't be a problem unless the tracts are much smaller than a CCD.
                     tract = skymap.findTract(wcs.pixelToSky(box.getCenter()))
-                    print("tract = ", tract)
```

Comment:
Yes, these removals were intentional. Part of this logging cleanup was removing unneeded output (as determined by me: jointcal's original authors left a lot of debugging statements in place).